### PR TITLE
Add linting and fix classlib indention

### PR DESCRIPTION
**Pull request by gusano**
This adds linting (for indention) as well as some indention fixes in the class library only.

~~Note that `SCDocRenderer.sc` is excluded as it was the only file which uses spaces there.~~

Travis runs the lint command and the build (on linux) will ~~eventually~~ fail fast if indention is broken in the class library.
(on travis linting is only run on linux because osx-ci uses bash-3 which does not support ** wildcard)

Re #2094

### CODE REVIEW

**Comment by crucialfelix** [comment]
Yay ! Thanks for doing this.  We should convert scdocrendere too. Is it because of html in the class file ?

**Comment by gusano** [comment]
> We should convert scdocrendere too. Is it because of html in the class file ?

@crucialfelix no, it appears to be the only file using spaces instead of tabs in the class library.
Gonna convert it as well then.

**Comment by gusano** [comment]
@crucialfelix done

**Comment by vivid-synth** [comment]
If anyone wants to review the non-whitespace changes, [here ya go](https://github.com/supercollider/supercollider/pull/2298/files?w=0)

**Comment by gusano** [comment]
@crucialfelix I'd be willing to add linting for at least CMakeLists.txt files as well but it's a bit messy right now so it might be better in another PR.

**Comment by crucialfelix** [comment]
yeah, let's do that later. and the cpp code too.

shit, this one already has a conflict.

I wanted to wait on it because the PRs that I'm merging now might cause a failure after they merge (and the linter is added).

File: SCClassLibrary/Common/Audio/BufIO.sc [hunk 1]
```diff
@@ -22,8 +22,8 @@ TGrains : MultiOutUGen {
 	*ar { arg numChannels, trigger=0, bufnum=0, rate=1, centerPos=0,
 			dur=0.1, pan=0, amp=0.1, interp=4;
 		if (numChannels < 2) {
-			 "TGrains needs at least two channels.".error;
-			 ^nil
+			"TGrains needs at least two channels.".error;
```
Comment (crucialfelix) [inline SCClassLibrary/Common/Audio/BufIO.sc]: this is the conflict. I just merged a pr that removes this if check

**Comment by gusano** [comment]
@crucialfelix I fixed the conflict and now the build is gonna fail fast (on linux) if there are any linting errors.

**Comment by crucialfelix** [comment]
I guess we should do this now. 

The only reason I was thinking to wait was because of other PR that will still be merged that might re-introduce spaces. If one does get through then master errors and we have to just jump in and fix up the spaces. No big deal.

**Pull request merged**